resources/privacy-and-gdpr.md

```diff
@@ -230,5 +230,5 @@ To update the SDK configuration, programmatically, use the following information
 ## Additional information
 
 * For more information about GDPR, see [GDPR and Your Business](https://www.adobe.com/privacy/general-data-protection-regulation.html)
-* To see the GDPR API documentation, go to [General Data Protection Regulation API](https://adobe.io/apis/cloudplatform/gdpr.html)
+* To see the Privacy Service API documentation, go to [Privacy Service API](https://experienceleague.adobe.com/docs/experience-platform/privacy/home.html)
```

Comment:
"Refer to the [Privacy Service API documentation] to learn how to programmatically manage privacy requests."

Should also update the link to point to the API guide rather than the home page: https://experienceleague.adobe.com/docs/experience-platform/privacy/api/getting-started.html